python/cudf/cudf/utils/ioutils.py

```diff
@@ -1430,12 +1434,15 @@
     better throughput by decomposing it and transferring multiple "blocks"
     in parallel (using a Python thread pool). Default allocation is
     {bytes_per_thread} bytes.
+expand_dir_pattern : str, default None
+    Glob pattern to use when expanding directories into file paths
+    (e.g. "*.json"). If this parameter is not specified, directories
+    will not be expanded.
 
 Returns
 -------
-filepath_or_buffer : str, bytes, BytesIO, list
-    Filepath string or in-memory buffer of data or a
-    list of Filepath strings or in-memory buffers of data.
+filepath_or_buffer : List[str, bytes, BytesIO]
```

Comment:
Is the suggestion to make this `List[str, bytes, BytesIO]`?

Reply:
Yup exactly. In general you don't need names when documenting what something return since (unlike a parameter) it has no name.

Reply:
This isn't critical obviously. Feel free to apply or not. Probably depending on whether or not Lawrence requests other changes since I don't think I have any.